File: lc1455.java
public class lc1455 {
    public static void main(String[] args) {
        // 1455. Check If a Word Occurs As a Prefix of Any Word in a Sentence
        String sentence = "i am tired", searchWord = "you";
        int ans=-1;
        String[] t = sentence.split(" ");
        for (int i=0;i<t.length;i++){
            if (t[i].length()>=searchWord.length()){
                if (t[i].startsWith(searchWord)){
                    ans=i+1;
                    break;
                }
            }
        }
        System.out.println(ans);
    }
}

File: lc1455.py
class Solution:
    def isPrefixOfWord(self, sentence: str, searchWord: str) -> int:
        a=list(sentence.split(" "))
        ans=-1
        for i,v in enumerate(a):
            if v.startswith(searchWord):
                ans=i+1
                break
        return ans
print(Solution.isPrefixOfWord(0,"i love eating burger","burg"))